docs/visualization/visualize_lfp_responses.ipynb

```diff
@@ -84,7 +84,7 @@
      "name": "stderr",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    # displays first 10 rows of the stim table you just retrieved
</pre><p>probably not necessary; the output will speak for itself here</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AllenInstitute/openscope_databook/pull/233/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>